### DISCUSSION TITLE
Tutorial: Deep dive into block styles

### DISCUSSION THREAD

**Discussion by justintadlock** [WordPress/developer-blog-content#41] [Topic Ideas]
Block styles, at least from what I've seen, have been one of the least used features for theme authors.  And, when they are used, they are underutilized, not showing off their full potential.

I'd like to do a deep dive into how to build them, the multiple methods of loading styles for them only when needed, and showcasing some advanced examples.

On top of this introduction post, I could also see this working as an ongoing series where we have contributing theme designers showcase new block style ideas.  And, because we have a good overview as a first post, posts in the series could link back to the first post to show the foundational pieces without having to re-teach it each time.

**Comment by bph** [comment]
I have been thinking about this as well. I still remember [Kjell Reigstad's WordCamp US 2021 talk](https://wordpress.tv/2021/12/24/kjell-reigstad-creative-use-of-block-styles/) - It was so inspiring!

**Comment by mburridge** [comment]
This tutorial could also cover using attributes instead of post meta as an alternative, more “block-friendly” approach.
